torch/nativert/executor/ExecutorConfig.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace torch::nativert {
+
+struct ExecutorConfig {
+  bool validateInputs = false;
+  bool debugNan = false;
+  bool enableStaticCPUKernels = false;
+  bool enableStaticMemoryPlanning = false;
+  bool runConstFolding = false;
+  // allows up to max number of concurrent threads.
+  int64_t maxNumConcurrentThreads = 8;
```

Comment:
Why not use the at flag we already have for this?

Reply:
https://github.com/pytorch/pytorch/blob/fb85ebd71028770df2be314ef8f95acc84306cd3/aten/src/ATen/Parallel.h#L16-L20